libbeat/docs/shared-ilm.asciidoc

```diff
@@ -0,0 +1,151 @@
+[[ilm]]
+[role="xpack"]
+== Set up index lifecycle management
+
+beta[]
+
+// REVIEWERS: should we mention that this requires a basic license? I started to
```

Comment:
I personally don't think we need it as it's the default we can expect and we have a pretty good error message. What are we doing in other docs parts?

Reply:
Agreed. I've set the xpack role on the docs, which means that the xpack label will show up (or whatever mechanism we decide to use in the future). I'm not sure we are completely consistent, but we typically only mention license levels when something higher than basic is required.